client/a8c-for-agencies/sections/marketplace/pressable-overview/plan-selection/details.tsx

```diff
@@ -130,7 +129,11 @@ export default function PlanSelectionDetails( {
 								: translate( 'Custom WordPress installs' ),
 							translate( '{{b}}%(count)s{{/b}} visits per month*', {
 								args: {
-									count: info ? formatNumber( info.visits ) : customString,
+									count: info
+										? numberFormat( info.visits, {
+												numberFormatOptions: { notation: 'compact' },
+										  } )
+										: customString,
```

Comment:
Same. Anyone can help me confirm this? 🙏

Reply:
Looks like this component is not being used anymore. We will remove this when we remove the feature flag logic.